### PR TITLE
Update w12-assignment-photo.tex

### PR DESCRIPTION
**Pull request by DagHemberg**
Changes made:

- Changed a doc link to reflect the latest version of Scala
- Added proper indentation for the class/object specifications
- Removed unnecessary semicolons in code specifications and examples
- Minor spelling/grammar fixes

### CODE REVIEW

File: compendium/modules/w12-assignment-photo.tex [hunk 1]
```diff
@@ -82,56 +82,56 @@ \subsection{Uppgiften}
 
 
 \code{ImageEditor}-klassen ska använda sig av en Stack[Image] för att hantera historiken. 
-Läs dokumentationen här: \url{https://www.scala-lang.org/api/2.13.6/scala/collection/mutable/Stack.html}
+Läs dokumentationen här: \url{https://www.scala-lang.org/api/3.0.1/scala/collection/mutable/Stack.html}
```
Comment (bjornregnell) [inline compendium/modules/w12-assignment-photo.tex:85]: Better use latest so we go with the version flow: 
https://www.scala-lang.org/api/current/scala/collection/mutable/Stack.html

File: compendium/modules/w12-assignment-photo.tex [hunk 2]
```diff
@@ -334,7 +334,7 @@ \subsection{Uppgiften}
 
 \subsection{Frivilliga extrauppgifter}
 
-\Task \textbf{Kontrastfilter.} Om man applicerar kontrastfiltrering på en färgbild så kommer bilden att konverteras till en gråskalebild. (Man kan naturligtvis förbättra kontrasten i en färgbild och få en färgbild som resultat. Då behandlar man de tre färgkanalerna var för sig.) Många bilder lider av alltför låg kontrast. Det beror på att bilden inte utnyttjar hela det tillgängliga området 0–255 för intensiteten. Man får en bild med bättre kontrast om man ''töjer ut'' intervallet enligt följande formel (linjär interpolation):
+\Task \textbf{Kontrastfilter.} Om man applicerar kontrastfiltrering på en färgbild så kommer bilden att konverteras till en gråskalebild (Man kan naturligtvis förbättra kontrasten i en färgbild och få en färgbild som resultat. Då behandlar man de tre färgkanalerna var för sig). Många bilder lider av alltför låg kontrast. Det beror på att bilden inte utnyttjar hela det tillgängliga området 0–255 för intensiteten. Man får en bild med bättre kontrast om man ''töjer ut'' intervallet enligt följande formel (linjär interpolation):
```
Comment (bjornregnell) [inline compendium/modules/w12-assignment-photo.tex:337]: As this is encapsulating two capitalized sentences the period should *inside* the parentheses.  And the preceding period should be added back.

**Comment by bjornregnell** [comment]
Thanks. I found two minor tweaks, see above. Do you want me to merge and fix it myself or do you have time to fix it today? Print deadline approaching...

**Review by bjornregnell — CHANGES_REQUESTED** [review]

File: compendium/modules/w12-assignment-photo.tex (hunk 2 above)
Comment (DagHemberg) [reply compendium/modules/w12-assignment-photo.tex:337]: Ah, I somehow missed the fact that there were 2 sentences in the parentheses. Will fix.

**Review by bjornregnell — APPROVED** [review]

**Pull request merged**